### PR TITLE
Add citations file and pipeline summary

### PR DESCRIPTION
**Pull request by drpatelh**
# nf-core/viralrecon pull request

Many thanks for contributing to nf-core/viralrecon!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/viralrecon branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/viralrecon)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/viralrecon/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CITATIONS.md [hunk 1]
```diff
@@ -1,28 +1,67 @@
...
 
-## Pipeline tools
+## Nextflow
 
 * [Nextflow](https://www.ncbi.nlm.nih.gov/pubmed/28398311/)
   > Di Tommaso P, Chatzou M, Floden EW, Barja PP, Palumbo E, Notredame C. Nextflow enables reproducible computational workflows. Nat Biotechnol. 2017 Apr 11;35(4):316-319. doi: 10.1038/nbt.3820. PubMed PMID: 28398311.
 
+## Pipeline tools
+
+* [ABACUS](https://www.ncbi.nlm.nih.gov/pubmed/19497936/)
```
Comment (saramonzon) [inline CITATIONS.md:10]: The name of the software is ABACAS! there is a tupo there!

File: README.md [hunk 2]
```diff
@@ -15,6 +15,27 @@ This pipeline is a re-implementation of the [SARS_Cov2_consensus-nf](https://git
 
 The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible.
 
+## Pipeline summary
+
+1. Raw read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+2. Adapter trimming ([`Trimmomatic`](http://www.usadellab.org/cms/?page=trimmomatic))
+3. Removal of host reads ([`Kraken2`](http://ccb.jhu.edu/software/kraken2/))
+4. Amplicon removal ([`iVar`](https://github.com/andersen-lab/ivar); *amplicon data only*)
```
Comment (saramonzon) [inline README.md:23]: It's primer removal, not amplicon removal, we remove the primers included in the reads from the amplicons.

File: README.md [hunk 3]
```diff
@@ -15,6 +15,27 @@ This pipeline is a re-implementation of the [SARS_Cov2_consensus-nf](https://git
...
+5. De novo assembly
+    1. Choice of multiple assemblers ([`SPAdes`](http://cab.spbu.ru/software/spades/), [`metaSPAdes`](http://cab.spbu.ru/software/meta-spades/), [`Unicycler`](https://github.com/rrwick/Unicycler))
+    2. Contiguate contigs assembly ([`ABACUS`](https://www.sanger.ac.uk/science/tools/pagit))
```
Comment (saramonzon) [inline README.md:26]: ABACAS again :)

**Comment by drpatelh** [comment]
Ok. Thanks 👍 Fixed https://github.com/nf-core/viralrecon/pull/23/commits/a87af4acb8a3afe26175da6feeb4c5ca28b3b436

**Pull request merged**